### DISCUSSION TITLE
安全設計 and 資訊安全計畫

### DISCUSSION THREAD

**Discussion by Willypai** [Willypai/devopsProject#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by H1s0kaaa** [comment]
Are we using any packages or api from other open source sites (except the NBA api)?

**Reply by b11009061** [reply]
We have tried other APIs, but we still think that the NBA api is easier to use.

**Comment by Willypai** [comment]
Enable the Dependabot alerts

**Reply by Willypai** [reply]
also w Dependabot security updates and Dependabot version updates

**Comment by H1s0kaaa** [comment]
Does our project involve any private data input from users? If not, I think it is not necessary for us to do data encryption.

**Reply by uhong0307** [reply]
Basically our system only get the live score of matches, so it is no too necessary to encrypt the data.

**Comment by H1s0kaaa** [comment]
What tool should we use for scanning dependencies? Whitesource or Dependabot?

**Reply by b11009061** [reply]
I think we can try dependabot first, as it seems that the set up process would take us less effort.
